customization/windows-template-customization.md

```diff
@@ -64,8 +64,8 @@ System properties
 
 Right click on computer and go to Properties > Advanced > Performances:
 
- * If your don't care about visual effect, in Visual Effect select "Adjust for best performance"
- * I personally tweak the page file size to win some place on my root.
+ * If you don't care about visual effect, in Visual Effect select "Adjust for best performance"
+ * I personally tweak the page file size to win some space on my root.
```

Comment:
Are you sure it's getting back some capacity that has been lost rather than a change to optimize for future filesystem behavior over time?